app/controllers/places_controller.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+class PlacesController < ApplicationController
+  def index
+    render locals: { places: Place.all }
+  end
+
+  def show
+    render locals: { place: }
+  end
+
+  def new
+    render locals: { place: Place.new }
+  end
+
+  def edit
+    render locals: { place: }
+  end
+
+  def create
+    place = Place.new(place_params.merge(account: current_account))
+    if place.save
+      raise
+      redirect_to places_path
+    else
+      render :new, locals: { place: }, status: :unprocessable_entity
+    end
+  end
+
+  def update
+    return redirect_to places_path if place.update!(place_params)
+
+    render 'edit', locals: { place: }
+  end
+
+  def place
+    @place ||= Place.find(params[:id])
```

Comment:
Keep in mind that we should not allow to get any Place via ID. Only places assigned to the account should be accessible here. So to the find you need to add as well `current_account` scope.